mailbox/api/src/main/java/org/apache/james/mailbox/events/MailboxListener.java

```diff
@@ -72,6 +117,10 @@ default boolean isHandling(Event event) {
         return true;
     }
 
+    default MaxQueueSize getMaxQueueSize() {
```

Comment:
Shouldn't this return Optional?

Reply:
We already have a `NONE`, I do not see what state would it reprensents

Reply:
(I don't like the null object pattern)